gunicorn/config.py

```diff
@@ -1049,12 +1048,12 @@ class AccessLogFormat(Setting):
         Identifier  Description
         ==========  ===========
         h           remote address
-        l           '-'
-        u           currently '-', may be user name in future releases
+        l           ``-``
```

Comment:
'-' means a string here, so \`\`'-'\`\` would be a better choice. See Python's codecs module documentation for a similar use case(e.g. \`\`'strict'\`\`).